Source/Core/AudioCommon/Mixer.h

```diff
@@ -42,7 +42,7 @@ class CMixer final
   float GetCurrentSpeed() const { return m_speed.load(); }
   void UpdateSpeed(float val) { m_speed.store(val); }
 private:
-  static constexpr u32 MAX_SAMPLES = 1024 * 4;  // 128 ms
+  static constexpr u32 MAX_SAMPLES = 1024 * 8;  // 256 ms
```

Comment:
This is likely only needed because of the former commit. We try to not get more than 40 ms.